make/manual

```diff
@@ -8,6 +8,25 @@ VERSION_STRING := $(MAJOR_VERSION).$(MINOR_VERSION).$(PATCH_VERSION)
 manual: $(STANAPI_HOME)src/docs/stan-reference/stan-reference.pdf
 	mkdir -p doc/
 	$(foreach artifact, $^, cp $(artifact) doc/$(basename $(notdir $(artifact)))-$(VERSION_STRING).pdf;)
+	echo '# This file is semicolon delimited' > doc/stan-functions.txt
```

Comment:
@bgoodri, mind if we put `@` before each of these lines to suppress the output to screen?